package.json

```diff
@@ -1,90 +1,32 @@
 {
   "name": "ember-amount-input",
-  "version": "5.0.0",
-  "description": "The default blueprint for ember-cli addons.",
-  "keywords": [
-    "ember-addon"
-  ],
-  "homepage": "https://qonto.github.io/ember-amount-input/versions/master",
+  "version": "0.0.0",
```

Comment:
We would like to keep versioning here

Reply:
Should the version be changed only in the addon? Or in both the addon and root package.json?

Reply:
Both, it's should be handle by the release process but we would like to keep versionning in both places